net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="79.10.2"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gwenhywfar$secondaryArchSuffix = $portVersion
+	cmd:gct_tool = $portVersion
+	cmd:gsa = $portVersion
+	cmd:gwbuild = $portVersion
+	cmd:gwenhywfar_config = $portVersion
+	cmd:mklistdoc = $portVersion
+	cmd:typemaker = $portVersion
+	cmd:typemaker2 = $portVersion
+	cmd:xmlmerge = $portVersion
+	lib:libgwenhywfar$secondaryArchSuffix = $libVersionCompat
+	lib:libgwengui_cpp$secondaryArchSuffix = $libVersionCompat
+	lib:libgwengui_gtk3$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libatk_1.0$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+   	lib:libgnutls$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libgtk_3$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+    "
+
+PROVIDES_devel="
+	gwenhywfar${secondaryArchSuffix}_devel = $portVersion
+	devel:libgwenhywfar$secondaryArchSuffix = $libVersionCompat
+	devel:libgwengui_cpp$secondaryArchSuffix = $libVersionCompat
+	devel:libgwengui_gtk3$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES_devel="
+	gwenhywfar$secondaryArchSuffix == $portVersion base
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+    "
+
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:gettext
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -if
+	runConfigure --omit-dirs "binDir" ./configure --bindir=$prefix/bin --with-guis=gtk3 LDFLAGS="-lnetwork"
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm $libDir/lib*.la
+
+	prepareInstalledDevelLibs \
+		libgwenhywfar \
+		libgwengui-cpp \
+		libgwengui-gtk3
+
+	fixPkgconfig
+
+	# devel package
+	packageEntries devel \
+		$developDir
```

Comment:
The config file (from $prefix/bin) should be added here, also the aclocal (from $dataDir) directory containing the m4 file, and the cmake directory from $libDir. :)

Reply:
Noted (see my reply on the previous one about gwenhywfar_config). I'll fix it as soon as I can.